src/stories/components/custom-table/custom-table.stories.tsx

```diff
@@ -1,36 +1,95 @@
+import { CustomTable2 } from './custom-table-2';
+import type { CustomTableColumn, CustomTableRow } from './custom-table-2';
+import type { SortEnum } from '@ses/core/enums/sort.enum';
+import type { ComponentMeta } from '@storybook/react';
 
 export default {
-  title: 'Components/General/CustomTable',
-  component: CustomTable,
-} as ComponentMeta<typeof CustomTable>;
+  title: 'Components/CuTable/CustomTable2',
+  component: CustomTable2,
+  parameters: {
+    layout: 'centered',
+    chromatic: {
+      viewports: [375, 834, 1194],
+      pauseAnimationAtEnd: true,
+    },
+  },
+} as ComponentMeta<typeof CustomTable2>;
 
-const Template: ComponentStory<typeof CustomTable> = (args) => (
-  <Provider store={store}>
-    <CustomTable {...args} />
-  </Provider>
-);
+const variantsArgs = [
+  {
+    columns: columns as CustomTableColumn[],
+    items: [
+      {
+        value: CoreUnit,
+        key: CoreUnit.shortCode,
+      },
+      {
+        value: CoreUnit,
+        key: CoreUnit.shortCode,
+      },
+      {
+        value: CoreUnit,
+        key: CoreUnit.shortCode,
+      },
+      {
+        value: CoreUnit,
+        key: CoreUnit.shortCode,
+      },
+      {
+        value: CoreUnit,
+        key: CoreUnit.shortCode,
+      },
+      {
+        value: CoreUnit,
+        key: CoreUnit.shortCode,
+      },
+      {
+        value: CoreUnit,
+        key: CoreUnit.shortCode,
+      },
+    ] as CustomTableRow[],
+    loading: false,
+    sortState: [] as SortEnum[],
+    handleSort: () => undefined,
+    headersSort,
+    // renderCard: (data: CustomTableRow, index: number) =>,
+    // renderCard?: (data: CustomTableRow, index: number) => JSX.Element //(data: CustomTableRow, index: number) => JSX.Element;
+    queryStrings: '',
+  },
+];
 
-export const Default = Template.bind({});
-Default.args = {
-  headers: ['Lorem', 'Ipsum', 'Dolor', 'Sit', 'Amet'],
-  items: [
-    ['Lorem', 'Ipsum', 'Dolor', 'Sit', 'Amet'],
-    ['Ipsum', 'Ipsum', 'Dolor', 'Sit', 'Amet'],
-    ['Lorem', 'Ipsum', 'Dolor', 'Sit', 'Amet'],
-    ['Lorem', 'Ipsum', 'Dolor', 'Sit', 'Amet'],
-    ['Lorem', 'Ipsum', 'Dolor', 'Sit', 'Amet'],
-    ['Lorem', 'Ipsum', 'Dolor', 'Sit', 'Amet'],
-    ['Lorem', 'Ipsum', 'Dolor', 'Sit', 'Amet'],
-  ],
-};
+export const [[Table, TableDark]] = createThemeModeVariants(CustomTable2, variantsArgs);
 
-export const Empty = Template.bind({});
-Empty.args = {
-  headers: ['Lorem', 'Ipsum', 'Dolor', 'Sit', 'Amet'],
-  items: [],
+Table.parameters = {
+  figma: {
+    component: {
+      375: 'https://www.figma.com/file/pyaYEjcwF2b5uf9y0vIfIy/SES-Dashboard?node-id=8868%3A143329&t=t4FrzPIVoc7xPgpL-4',
+      834: 'https://www.figma.com/file/pyaYEjcwF2b5uf9y0vIfIy/SES-Dashboard?node-id=5232%3A93999&t=t4FrzPIVoc7xPgpL-4',
+      1194: {
+        component:
+          'https://www.figma.com/file/pyaYEjcwF2b5uf9y0vIfIy/SES-Dashboard?node-id=8868%3A139602&t=BJYSrpHcrXx2uY02-4',
```

Comment:
These breakpoints should include a component style to fix the width according to the image width so the image match perfectly